### DISCUSSION TITLE
PiPedal 1.2.44 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#183] [Releases]
This version includes the following new features:

- Double-tap (or double-click) dial controls to reset them to default values.
- Update PiPedal to the latest version from within the PiPedal web UI or PiPedal client. PiPedal monitors the PiPedal github repository for new releases,
  and prompts you to update when new releases become available.
- Choose whether to monitor Release, Beta or Developer update streams, or disable update monitoring altogether.
- All PiPedal configuration settings now preserved when upgrading.
- Advanced configuration options are now stored in /var/pipedal/config/config.json so that they are preserved when upgrading.

Bug fixes:

- Fixed a bug which prevents the PiPedal UI from updating properly when using TooB Convolution Reverb and CabIR effects.
- Plugin controls now work when there's no audio device running, or when the audio device stops.
- Minor style and theming issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/v1.2.44'>PiPedal 1.2.44 Release</a>.</em>

**Comment by 38github** [comment]
That is a bunch of great features! Double-tap to reset is something I will enjoy. The updating in the GUI but also able to choose branch is great!

Your absence seems to have given you a lot of will power now that you have returned! I will recommend PiPedal to friends and internet acquaintances over MOD/MODEP.
